poc/idpf.sage

```diff
@@ -117,6 +119,31 @@ def test_idpf(Idpf, alpha, level, prefixes):
                 alpha, prefix, level, got, want))
 
 
+def gen_test_vec(Idpf, alpha, test_vec_instance):
+    beta_inner = [[Idpf.FieldInner(1)] * Idpf.VALUE_LEN] * (Idpf.BITS-1)
+    beta_leaf = [Idpf.FieldLeaf(1)] * Idpf.VALUE_LEN
+    (public_share, keys) = Idpf.gen(alpha, beta_inner, beta_leaf)
+
+    printable_beta_inner = [
+        [ elem.as_unsigned() for elem in value ] for value in beta_inner
+    ]
+    printable_beta_leaf = [ elem.as_unsigned() for elem in beta_leaf ]
+    printable_keys = [ key.hex() for key in keys ]
+    test_vec = {
+        'beta_inner': printable_beta_inner,
+        'beta_leaf': printable_beta_leaf,
+        'public_share': public_share.hex(),
+        'keys': printable_keys,
+    }
```

Comment:
The test vector should include alpha as well, so that it's self-contained. (Perhaps as an array of `0` and `1`?)

Reply:
Are you alright with an unsinged integer here? From the spec perspective, this is the most natural way to represent `alpha`.

Reply:
If we included `bits` as a number and `alpha` as a number-as-a-string, that would work.

Reply:
Done.